src/main/java/uk/gov/justice/digital/domain/model/TableInfo.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.domain.model;
+
+import lombok.Data;
+
+@Data
+public class TableInfo {
+    protected String prefix;
+    protected String schema;
+    protected String table;
+
+    protected TableInfo(final String prefix, final String schema, final String table) {
```

Comment:
Using the `@Data` annotation you get a constructor for free so this can be removed.